.goreleaser.yml

```diff
@@ -0,0 +1,33 @@
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+    binary: crd2pulumi
+    main: ./cmd/root.go
```

Comment:
Not 100% sure on this, but shouldn't this be `./main.go` instead?

Reply:
ah you are correct! Good spot